packages/cli/src/config/schema.ts

```diff
@@ -989,6 +989,12 @@ export const schema = {
 					doc: 'Access secret in S3-compatible external storage',
 				},
 			},
+			skipPruningRequests: {
+				format: Boolean,
+				default: false,
+				env: 'N8N_EXTERNAL_STORAGE_S3_SKIP_PRUNING_REQUESTS',
+				doc: 'Skip requests sent by n8n to S3 to remove binary data files during pruning. Enable this setting if you have set a bucket-level TTL, to minimize requests to S3.',
```

Comment:
Do we really need an additional flag for this? 
Boolean flags make sense when we want to support both behaviors.
In this case, could we instead just not prune data on S3, and recommend setting up lifecycle policies on the bucket? 
Object stores natively do a way better job of pruning old data than we'll ever do.

Reply:
We discussed with @krynble to make this optional but I can change it. Let me know.